docs/reference/settings/security-settings.asciidoc

```diff
@@ -2174,7 +2174,29 @@ rotated the signing keys.
 +
 File-based resources are polled at a frequency determined by the global {es}
 `resource.reload.interval.high` setting, which defaults to `5s`.
-// end::jwt-jwkset-path-tag[]
+// end::jwt-pkc-jwkset-path-tag[]
+
+// tag::jwt-hmac-jwkset-tag[]
+`hmac_jwkset` {ess-icon}::
+(<<secure-settings,Secure>>)
+The file contents of a JSON Web Key Set with the secret key material that
+the JWT Realm uses for verifying token signatures.
+This format is compatible with multiple keys and optional attributes.
+Only one of `hmac_jwkset` or `hmac_key` can be used simultaneously.
+Using the HMAC JWK Set is preferred.
+// end::jwt-hmac-jwkset-tag[]
+
+// tag::jwt-hmac-key-tag[]
+`hmac_key` {ess-icon}::
+(<<secure-settings,Secure>>)
+The contents of a JSON Web Key with the secret key material that
```

Comment:
The `hmac_jwkset` contains "The file contents of  JWK Set", whereas the `hmac_key` indicates only "The contents". Is the `hmac_key` also a file? I'm trying to better understand the distinction between these settings other than what they support.

Reply:
Both can be `The contents`, so remove `file`. I was implying they would be loaded via the different elasticsearch-keystore commands listed in the `[JWT authentication](https://elasticsearch_85189.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/jwt-realm.html)` page.

HMAC JWKSet loaded from a file
`bin/elasticsearch-keystore add-file xpack.security.authc.realms.jwt.jwt1.hmac_jwkset <path>`

HMAC Key loaded from a string on the command line
`bin/elasticsearch-keystore add xpack.security.authc.realms.jwt.jwt1.client_authentication.shared_secret`